EIPS/eip-2982.md

```diff
@@ -0,0 +1,197 @@
+
+___A hard fork of the eth2 beacon chain is planned for 5 months after genesis. This hard fork will adjust `INACTIVITY_PENALTY_QUOTIENT`, `PROPORTIONAL_SLASHING_MULTIPLIER`, and `MIN_SLASHING_PENALTY_QUOTIENT` to their final, more secure values.___
+
+## Rationale
+
+Ethereum blocks are consistently full due to increasingly high demand for decentralized applications. Ever since the first serious spikes in adoption in 2017 (cryptokitties), the Ethereum community has consistently and vocally demanded scaling solutions.
+
+Since day 0 of Ethereum, the investigation and expectation in scaling solutions has been two-pronged -- scale from both Layer 1 upgrades and Layer 2 adoption.
+
+For details regarding specification design rationale and decisions, see the following:
+* [Serenity Design Rationale](https://notes.ethereum.org/@vbuterin/rkhCgQteN)
+* [Ben Edgington's Eth2 Annotated Spec](https://benjaminion.xyz/eth2-annotated-spec/)
+* [Vitalik Buterin's Eth2 Annotated Spec](https://github.com/ethereum/annotated-spec/blob/master/phase0/beacon-chain.md)
+
+## Backwards Compatibility
+
+Although this EIP does not introduce any immediate changes to the current Ethereum mainnet, this EIP lays the groundwork for future backwards incompatibilities through the introduction of the new eth2 consensus mechanism in which Ethereum will be integrated in subsequent phases. To secure this mechanism, users move ether into the beacon chain and additional ether is issued. This EIP is a commitment to this path being canonical, as well as directly informing the future and roadmap of Ethereum mainnet.
+
+## Implementation
+
+The following implementations of eth2 Phase 0 exist. At the time of writing at least five are near production-ready and in testnet/audit phase:
+
+* [Cortex](https://github.com/NethermindEth/cortex) (.net)
+* [Lighthouse](https://github.com/sigp/lighthouse) (rust)
+* [Lodestar](https://github.com/ChainSafe/lodestar) (javascript)
+* [Nimbus](https://github.com/status-im/nim-beacon-chain) (nim)
+* [Prysm](https://github.com/prysmaticlabs/prysm/) (go)
+* [Teku](https://github.com/pegasyseng/teku) (java)
+* [Trinity](https://github.com/ethereum/trinity/) (python)
+
+In addition to implementations of the spec, the deposit contract is [implemented in Solidity](), along with a [formal verification](https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/deposit-formal-verification.pdf) of the bytecode.
+
+## Security Considerations
+
+Eth2 is a major overhaul of the Ethereum's core consensus from PoW to a sharded PoS. There are inherent risks in this migration but there is a deep canon of research literature analyzing security and trade-offs. _The following only represents a high level selection of the resources available:_
```

Comment:
```suggestion
Eth2 would be a major overhaul of the Ethereum's core consensus from PoW to a sharded PoS. There are inherent risks in this migration but there is extensive research literature analyzing security and trade-offs. _The following only represents a high level selection of the resources available:_
```
More discussion of the risks would be helpful here.  And "canon" I think more often refers to scriptures.